<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic File Renamer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QToolButton" name="btnSelectDir">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="inPattern">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>%P.S%SE%E</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnLoad">
      <property name="text">
       <string>&amp;Load</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Extensions:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use named capture groups , for the pattern (?&amp;lt;capturename&amp;gt;XXXXX) for the capture group, in the output use &amp;lt;capturename&amp;gt;&lt;/p&gt;&lt;p&gt;&amp;lt;program&amp;gt;: Program Title&lt;br/&gt;&amp;lt;season&amp;gt;: Season Number (Don't include S in captured text)&lt;br/&gt;&amp;lt;episode&amp;gt;: Episode Number (Don't Include E in captured text)&lt;br/&gt;&amp;lt;title&amp;gt;: Episode Title&lt;br/&gt;&lt;br/&gt;(XXXX):&amp;lt;capturename&amp;gt; sets XXX to replace capture group &amp;lt;capturename&amp;gt;&lt;br/&gt;&lt;br/&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QTreeView" name="files">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pattern:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Directory:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="outPattern">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>%P - S%SE%E</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="directory">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="extensions">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>*.mkv;*.mp4;*.avi;*.idx;*.sub</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnTransform">
      <property name="text">
       <string>Transform</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="btnSaveM3U">
      <property name="text">
       <string>Save M3U</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetDirectory">
   <property name="text">
    <string>Set &amp;Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>directory</tabstop>
  <tabstop>btnSelectDir</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>extensions</tabstop>
  <tabstop>inPattern</tabstop>
  <tabstop>outPattern</tabstop>
  <tabstop>btnTransform</tabstop>
  <tabstop>btnSaveM3U</tabstop>
  <tabstop>files</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
